### PR TITLE
incorporate method line info

### PR DESCRIPTION
**Pull request by LuxiWang99**
iss#26

### CODE REVIEW

**Comment by LuxiWang99** [comment]
Also added a switch to delete/keep the tempfiles. 
Should probably document it after the code is reviewed.

**Comment by izgzhen** [comment]
```
Conflicting files
build-run-markii.sh
```

@LuxiWang99

**Comment by izgzhen** [comment]
You need to rebase

File: markii [hunk 1]
```diff
@@ -199,6 +200,12 @@ def main():
                                 required=True,
                                 help='path to the APK')
 
+    parser_analyze.add_argument('-temp',
+                                dest='remove_temp',
+                                required=False,
+                                default='F',
+                                help='rempve temp file (T/F)')
```
Comment (izgzhen) [inline markii]: typo

File: src/main/scala/com/research/nomad/markii/GUIAnalysis.scala [hunk 2]
```diff
@@ -164,10 +174,16 @@ object GUIAnalysis extends IAnalysis {
     }
     for ((eventType, methodName) <- viewNode.getInlineClickHandlers) {
```
Comment (izgzhen) [inline src/main/scala/com/research/nomad/markii/GUIAnalysis.scala]: update `methodName` to `methodInfo`

File: src/main/scala/com/research/nomad/markii/GUIAnalysis.scala [hunk 3]
```diff
@@ -128,6 +128,16 @@ object GUIAnalysis extends IAnalysis {
     printWriter.close()
   }
 
+  private def getJavaLineNumber(sootMethod:SootMethod):Int= {
```
Comment (izgzhen) [inline src/main/scala/com/research/nomad/markii/GUIAnalysis.scala]: nit: try to follow the style of existing code

**Review by izgzhen — CHANGES_REQUESTED** [review]

File: src/main/scala/com/research/nomad/markii/GUIAnalysis.scala (hunk 3 above)
Comment (LuxiWang99) [reply src/main/scala/com/research/nomad/markii/GUIAnalysis.scala]: Can you be more specific on this comment?

Comment (izgzhen) [reply src/main/scala/com/research/nomad/markii/GUIAnalysis.scala]:
```scala
  private def getJavaLineNumber(sootMethod: SootMethod): Int = {
```

would be better

Comment (LuxiWang99) [reply src/main/scala/com/research/nomad/markii/GUIAnalysis.scala]: No problem

**Review by izgzhen — APPROVED** [review]

**Pull request merged**